src/script/components/Modals/PrimaryModal/PrimaryModal.tsx

```diff
@@ -55,7 +55,7 @@ export const PrimaryModalComponent: FC = () => {
   const isModalVisible = currentId !== null;
   const passwordValueRef = useRef<HTMLInputElement>(null);
   const [isFormSubmitted, setIsFormSubmitted] = useState(false);
-  const isBackupPasswordValid = useMemo(() => passwordInput && isValidPassword(passwordInput), [passwordInput]);
+  const isBackupPasswordValid = useMemo(() => passwordInput === '' || isValidPassword(passwordInput), [passwordInput]);
```

Comment:
Just curious,
Can we move this logic to `isValidPassword` function and pass an option to allow empty values?

Reply:
I don't think it's a good idea, the `isValidPassword` function is a generic util, and how the backup password works now is pretty unusual, maybe we're gonna change this behavior in the future

Besides that, IMHO it's beyond the responsibility of this function - we don't need to validate the password if it doesn't exist 🤷‍♂️